analyses/molecular-subtyping-MB/README.md

```diff
@@ -143,14 +143,7 @@ For each sample_id with multiple RNA samples, the following logic is implemented
 - IF the RNA specimen with mismatched classifications includes a match to the subtype of the consensus of the other RNA specimen,
 - THEN propagate the subtype to the second RNA specimen.
 
-For both the consensus corrected output and consensus uncorrected output, 25/32 match with the reported subtype so the % accuracy is the same i.e. 78.125%. Between the two consensus outputs, there is an overlap of 24/25 matched subtypes. The following samples are mismatched between the two consensus outputs:
-
-BS_HB03GSHF is correctly predicted by consensus batch-corrected output but not by uncorrected output:
-
-| Consensus Output | Kids_First_Biospecimen_ID | pathology_subtype | MM2S_best_fit | medulloPackage_best_fit | molecular_subtype | match |
-|------------------|---------------------------|-------------------|---------------|-------------------------|-------------------|-------|
-| Corrected        | BS_HB03GSHF               | WNT               | WNT           | WNT                     | WNT               | TRUE  |
-| Uncorrected      | BS_HB03GSHF               | WNT               | SHH           | WNT                     | NA                | NA    |
+For the consensus corrected output, 29/43 (67.44%) and for the consensus uncorrected output, 30/43 (69.76%) match with the reported subtype. The following table shows the difference between the two outputs:
```

Comment:
@komalsrathi is this still correct? It seems to be that if each individual classifier has 90 or 95% accuracy alone, that we should not see such a drastic drop when combining them.

Reply:
@jharenza thanks for catching that - I forgot to update the README.